cmd/pj-rehearse/main.go

```diff
@@ -140,26 +152,26 @@ func main() {
 		clusterConfig, err = loadClusterConfig()
 		if err != nil {
 			logger.WithError(err).Error("could not load cluster clusterConfig")
-			gracefulExit(o.noFail, misconfigurationOutput)
+			gracefulExit(o.noFail, misconfigurationOutput, nil)
 		}
 	}
 
 	prConfig := config.GetAllConfigs(o.releaseRepoPath, logger)
 	masterConfig, err := config.GetAllConfigsFromSHA(o.releaseRepoPath, jobSpec.Refs.BaseSHA, logger)
 	if err != nil {
 		logger.WithError(err).Error("could not load configuration from base revision of release repo")
-		gracefulExit(o.noFail, misconfigurationOutput)
+		gracefulExit(o.noFail, misconfigurationOutput, nil)
 	}
 
 	// We always need both Prow config versions, otherwise we cannot compare them
 	if masterConfig.Prow == nil || prConfig.Prow == nil {
 		logger.WithError(err).Error("could not load Prow configs from base or tested revision of release repo")
-		gracefulExit(o.noFail, misconfigurationOutput)
+		gracefulExit(o.noFail, misconfigurationOutput, nil)
 	}
 	// We always need PR versions of templates and ciop config, otherwise we cannot provide them to rehearsed jobs
 	if prConfig.Templates == nil || prConfig.CiOperator == nil {
 		logger.WithError(err).Error("could not load template/ci-operator configs from tested revision of release repo")
-		gracefulExit(o.noFail, misconfigurationOutput)
+		gracefulExit(o.noFail, misconfigurationOutput, nil)
```

Comment:
nit: if you find yourself making one call with a new argument and N with `nil`, make a new function:


```go
gracefulExitWithConfigMapCleanup(o.noFail, configmapOutput, controller)
```

(not that this is the best solution for this specific instance, but just as a nit